src/uploader.js

```diff
@@ -154,12 +154,11 @@ export default class Uploader {
           formData.append(name, value);
         });
       }
-
       upload = ajax.post({
         url: this.config.endpoints.byFile,
         data: formData,
-        type: ajax.contentType.JSON,
-        headers: this.config.additionalRequestHeaders
+        type: ajax.contentType.FORM,
```

Comment:
we can remove `type` param. `ajax.contentType.FORM` will be used automatically if `data` is a FormData object. [see source](https://github.com/codex-team/ajax/blob/master/src/index.js#L438)

```suggestion
```

Reply:
It's not working :(. Uploading by file works but it stops working after uploading a file by url (the contentType does not change, it remains as _json_). Maybe a bug with that feature in *ajax* module?